api/src/main/java/org/openmrs/module/fhir2/api/impl/FhirConditionServiceImpl.java

```diff
@@ -23,19 +23,19 @@
 
 @Component
 public class FhirConditionServiceImpl implements FhirConditionService {
```

Comment:
Remove this file from your PR

Reply:
This pr was mistakenly added let me kindly get rid out of it thanks